cloudsql/postgresql/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "appengine-cloudsql-postgres",
+  "description": "Node.js PostgreSQL sample for Cloud SQL on App Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "lint": "repo-tools lint",
```

Comment:
Please remove `lint` and `pretest`. Run `npm run lint` in the parent directory, and `npm run lint -- --fix` if needed.

Reply:
Done.